packages/lesswrong/server/recommendations.ts

```diff
@@ -167,12 +167,12 @@ const allRecommendablePosts = async ({currentUser, algorithm}: {
 }): Promise<Array<DbPost>> => {
   if (Posts.isPostgres()) {
     const joinHook = algorithm.onlyUnread && currentUser
-      ? `LEFT JOIN "ReadStatuses" rs ON rs."postId" = "Posts"._id AND rs."userId" = '${currentUser._id}' AND rs."isRead" = FALSE`
+      ? `LEFT JOIN "ReadStatuses" rs ON rs."postId" = "Posts"._id AND rs."userId" = '${currentUser._id}' AND rs."isRead" IS TRUE` // only include the isRead = true rows so we can filter them out via null comparison
```

Comment:
This comment refers to a separate bug where doing `isRead <> TRUE` in the query gives a different result to `isRead is not TRUE` (`is not TRUE` gives the correct result). `<> TRUE` excludes nulls for some reason (see here for the [reason](https://stackoverflow.com/questions/3777230/is-there-any-difference-between-is-null-and-null))